### PR TITLE
fix group collide bug

### PR DESCRIPTION
**Pull request by dabing1022**
In Chipmunk2D,  objects in the same non-zero group won't collide.

### CODE REVIEW

File: cocos/physics/CCPhysicsWorld.cpp [hunk 1]
```diff
@@ -275,7 +275,7 @@ int PhysicsWorld::collisionBeginCallback(PhysicsContact& contact)
     
     if (shapeA->getGroup() != 0 && shapeA->getGroup() == shapeB->getGroup())
     {
-        ret = shapeA->getGroup() > 0;
+        ret = false;
```
Comment (boyu0) [inline cocos/physics/CCPhysicsWorld.cpp:278]: Yes, But this value is designed the same as box2d, that's why we didn't use cpSetGroup to set this value.

Comment (dabing1022) [inline cocos/physics/CCPhysicsWorld.cpp:278]: It seems no sense. PhysicsWorld is constructed by Chipmunk2D essentially, we should keep the same mechanism as Chipmunk2D, not box2D.

------------------ 原始邮件 ------------------
发件人: "boyu0"; 
发送时间: 2014年4月6日(星期天) 上午9:05
收件人: "cocos2d/cocos2d-x"; 
抄送: "ChildhoodAndy"; 
主题: Re: [cocos2d-x] fix group collide bug (#6148)

In cocos/physics/CCPhysicsWorld.cpp:

Comment (boyu0) [inline cocos/physics/CCPhysicsWorld.cpp:278]: @dabing1022 
I don't think so, why does someone use our physics integration have to care about which physics engine they use? We only care about API friendly, performance and functionality, but not which physics engine they use, and which mechanism has to be.

Comment (boyu0) [inline cocos/physics/CCPhysicsWorld.cpp:278]: We use this mechanism is to make friendly with someone has SpriteKit experience, and this reason make more sense to me.

**Comment by dabing1022** [comment]
Well, OK. Thanks for your explain, @boyu0 . We can close this issue now:)

**Pull request closed without merging**